cogs/bot_commands.py

```diff
@@ -40,9 +39,7 @@ async def read_standings(self, ctx):
                     await ctx.send(file=discord.File('image-displayed.png'))
                     await ctx.send(file=discord.File('output.csv'))
                     description = (f"Google sheet copy is available here: {config('DOCS_LINK')}" +
-                                   "\n\nYou can copy paste this into the data collection sheet." +
-                                   "\nNOTE: I may have missed the highlighted name in the image."
```

Comment:
removing old and outdated error message that no longer holds as this issue has since been resolved.